docker/docker-compose.yaml

```
@@ -19,6 +19,12 @@ volumes:
       type: none
       device: ${PANDIR}/huntsman-config/conf_files/pocs
       o: bind
+  archive:
```

```suggestion
  archiver:
```
to be consistent with the other yaml parameters?

---

This corresponds to the archive directory and is not directly related to the archiver class